src/NzbDrone.Core/Indexers/Nyaa/NyaaSettings.cs

```diff
@@ -25,8 +25,8 @@ public class NyaaSettings : ITorrentIndexerSettings
 
         public NyaaSettings()
         {
-            BaseUrl = "http://www.nyaa.se";
-            AdditionalParameters = "&cats=1_37&filter=1";
+            BaseUrl = "http://www.nyaa.si";
```

Comment:
I’d just wipe out the preset url and make it a “”, Nyah changes so much no reason to keep changing it.. I think Sonarr has done the same.

Reply:
Presets removed